File: 485_Max_Consecutive_One/Max_Consecutive_One.go
func findMaxConsecutiveOnes(nums []int) int {
    var max, cnt int = 0, 0
    for _, d := range nums {
        if d == 1 {
            cnt += 1
        }else {
            cnt = 0
        }
        if cnt > max {
            max = cnt
        }
    }
    return max
}

File: 485_Max_Consecutive_One/Max_Consecutive_One.py
class Solution(object):
    def findMaxConsecutiveOnes(self, nums):
        """
        :type nums: List[int]
        :rtype: int
        """
        cnt, max_cnt = 0, 0
        for n in nums:
            if n == 1:
                cnt += 1
            else:
                cnt = 0
            if cnt > max_cnt: max_cnt = cnt
        return max_cnt
